packages/did-comm/src/didcomm.ts

```diff
@@ -603,7 +603,9 @@ export class DIDComm implements IAgentPlugin {
         }
 
         debug('Sending to %s', serviceEndpoint)
-        const res = await fetch(serviceEndpoint, {
+        const endpointUri = (typeof serviceEndpoint === 'string' || serviceEndpoint instanceof String) ? serviceEndpoint : serviceEndpoint.uri
```

Comment:
is this OR really needed here?
` || serviceEndpoint instanceof String`

It's not a request to change, just trying to learn in case there is some corner case that I'm not aware of.

Reply:
no you're probably right. I had seen this pattern somewhere, but we don't use it throughout our project and I think it's unnecessary